src/Components/Nav/Nav.js

```diff
@@ -1,9 +1,167 @@
-import React, { Component } from "react";
+import React from "react";
+import styled from "styled-components";
+import "../../Styles/common.scss";
 
-class Nav extends Component {
-  render() {
-    return <div></div>;
-  }
+export default function Nav() {
+  return (
+    <NavContents className="Nav">
+      <NavWrapper>
+        <div>
+          <NavLeftButton>
+            <li>
+              <KinfolkButton>Subscribe</KinfolkButton>
+            </li>
+            <li>Issue 36</li>
+            <li>Shop</li>
+          </NavLeftButton>
+        </div>
+        <TitleLogo>
+          <img alt="KinfolkLogo" src={LogoURL} />
+        </TitleLogo>
+        <NavRightButton>
+          <li>
+            <CartMark>
+              <svg
+                viewBox="0 0 30 30"
+                width="55px"
+                height="55px"
+                role="presentation"
+              >
+                <rect x="1" y="5" width="20px" height="13px"></rect>
+                <polyline points="5 7 5 1 17 1 17 7"></polyline>
+              </svg>
+              <div>
+                <span>(1)</span>
+              </div>
+            </CartMark>
+          </li>
+          <li>
+            <SearchMark>
+              <svg
+                fill="#000000"
+                viewBox="0 0 30 30"
+                width="40px"
+                height="40px"
+              >
+                <path d={SearchMarkPath} />
+              </svg>
+            </SearchMark>
+          </li>
+          <li>
+            <BurgerMark>
+              <span></span>
+              <span></span>
+              <span></span>
+            </BurgerMark>
+          </li>
+        </NavRightButton>
+      </NavWrapper>
+    </NavContents>
+  );
 }
 
-export default Nav;
+const NavContents = styled.nav`
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  padding: 0 3.5%;
+`;
+
+const NavWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 25px 0;
+  border-bottom: 2px solid black;
+`;
+
+const KinfolkButton = styled.a`
```

Comment:
```suggestion
const KinfolkButton = styled(Link)`
```
링크 컴포넌트 상속받아서 스타일링 해주세요

Reply:
넵넵!!🙌🏻